README.md

```diff
@@ -56,7 +56,7 @@ npm install --save fetch-redux-crud
 import { config } from 'fetch-redux-crud'
 
 config.apiUrl = 'http://path.to.api.domain/api/v1'
-config.loginUrl = 'http://path.to.login/page'
+config.redirectUrl = 'http://path.to.login/page'
```

Comment:
Was just making the rest of the app match https://github.com/nano3labs/fetch-redux-crud/blob/93e215da0fcdc6d80a96e43af6a056959b83d853/src/settings.js#L3

Reply:
Since this is a general purpose library though, I think we should not have this built-in and leave it up to the user

Reply:
yes I agree, we should provide hooks for callbacks where possible and leave it up to the user to implement

Reply:
ok I'll accept your PR with this change and increment the version to 0.1.0 since this is a breaking change. I'd like to remove it and go to callbacks instead in 0.2.0 as you said